content/python/concepts/PIP/pip.md

```diff
@@ -0,0 +1,50 @@
+---
+Title: ''
+Description: ''
```

Comment:
According to the [Contribution Guide,](https://github.com/Codecademy/docs/blob/main/.github/CONTRIBUTING.md) `Description` is missing which caused the test to fail.